addons/account/models/account_move.py

```diff
@@ -889,9 +893,19 @@ def _compute_name(self):
             if move.date and (not move_has_name or not move._sequence_matches_date()):
                 move._set_next_sequence()
 
-        self.filtered(lambda m: not m.name and not move.quick_edit_mode).name = '/'
+        self.filtered(lambda m: not m.name and not move.quick_edit_mode and move._get_last_sequence()).name = '/'
         self._inverse_name()
 
+    @api.depends('journal_id', 'sequence_number', 'sequence_prefix')
+    def _compute_name_placeholder(self):
+        for record in self:
+            record.name_placeholder_list = "/"
+            if record._get_last_sequence() or record.posted_before or record.state != "draft":
+                record.name_placeholder_form = False
+                record._compute_name()
+            else:
+                record.name_placeholder_form = record._get_starting_sequence()[:-1] + "1"
```

Comment:
This is not correct if there is a `sequence_override_regex` set on the journal because there could be a suffix after the sequence number